Makefile

```diff
@@ -24,7 +24,7 @@
 # put the top level directory of the Installation here
 # ( try "pwd" if in doubt)
 #
-TOPDIR=
+TOPCARATDIR=
```

Comment:
Of course this line now means that those compiling carat locally will also have to change `TOPCARATDIR` to `TOPDIR`. I hope we may have `configure` soon, so this is a temporary measure.